src/main/java/frc/robot/RobotConstantsIO.java

```diff
@@ -0,0 +1,14 @@
+package frc.robot;
+
+import frc.robot.util.TunableNumber;
+
+public interface RobotConstantsIO {
+  public static TunableNumber kDriveP = null;
+  public static TunableNumber kDriveI = null;
+  public static TunableNumber kDriveD = null;
+  public static TunableNumber kTurningP = null;
+  public static TunableNumber kTurningI = null;
+  public static TunableNumber kTurningD = null;
+  public static double kMaxSpeedMetersPerSecond = 0;
+  public static double kMaxAngularRotationsPerSecond = 0;
+}
```

Comment:
Not sure how I like this approach. Statics can't be overwritten in subclasses. At least they're not meant to be. We should seriously consider moving mk2 to its own repo or very least branch. It is just the drivebase, so once we have it coded up, we shouldn't need to be concerned with anything else. Instead we should just have the constants for the comp bot